cvmfs/util.cc

```diff
@@ -755,12 +755,13 @@ bool GetUidOf(const std::string &username, uid_t *uid, gid_t *main_gid) {
 bool GetGidOf(const std::string &groupname, gid_t *gid) {
   char buf[16*1024];
   struct group grp;
-  struct group *result;
-  getgrnam_r(groupname.c_str(), &grp, buf, sizeof(buf), &result);
-  if (result == NULL)
-    return false;
-  *gid = result->gr_gid;
-  return true;
+  struct group *result = NULL;
+  if (getgrnam_r(groupname.c_str(), &grp, buf, sizeof(buf), &result) == 0 &&
```

Comment:
According to the documentation:

_On success, getgrnam_r() and getgrgid_r() return zero, and set *result to grp. If no matching group record was found, these functions return 0 and store NULL in *result. In case of error, an error number is returned, and NULL is stored in *result._

Which means you are right, but we do not exhaustively check everything. On the other hand, the code is actually more readable...
I guess there is no problem if we gain more readability in exchange, so I'd better change it back